web_autoflip_demo/video_upload/video_upload.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+*/
+
+//check if browser supports file api and filereader features
+if (window.File && window.FileReader && window.FileList && window.Blob) {
+  // this function is called when the input loads a video
+  function showVideo(): void {
+    var file = (<HTMLInputElement>event.target).files[0];
```

Comment:
Aligning with google style guide don't use var, use const with variables that aren't reassigned.

Reply:
fix